sway-launcher-desktop.sh

```diff
@@ -153,7 +162,10 @@ desktop)
 
                 END{
                   if(path){
-                    print "cd " path " &&"
+                    printf "cd " path " &&"
+                  }
+                  if (terminal){
+                    printf terminal_command " "
```

Comment:
Why printf?

Reply:
Because `print` appends a newline which makes the command (for ranger):

```
termite -e
ranger
```

Which doesn’t work.

Reply:
I see. Makes sense